spark/src/main/kotlin/com/adevinta/spark/components/toggles/Switch.md

```diff
@@ -41,6 +41,18 @@ SwitchLabelled(
 ) { Text(text = "Switch On") }
 ```
 
+### Styles
+
+The `Switch` and `SwitchLabelled` accept 7 [ToggleIntent](ToggleIntent.kt)s:
+
+- Primary (default)
```

Comment:
Isn't the `Basic` intent the default one now ?

Reply:
Ah yes, good catch. I lost this change in some complicated rebase